config/rubocop/naming.yml

```diff
@@ -13,12 +13,9 @@ Naming/MethodName:
   Exclude:
     - 'app/models/accrediting_provider_enrichment.rb'
 
+# Allow numbers in variable names
 Naming/VariableNumber:
-  Exclude:
-    - 'app/serializers/api/public/v1/serializable_location.rb'
-    - 'app/serializers/api/public/v1/serializable_provider.rb'
-    - 'spec/serializers/api/public/v1/serializable_location_spec.rb'
-    - 'spec/serializers/api/public/v1/serializable_provider_spec.rb'
+  Enabled: false
```

Comment:
Did this cause a lot more complaints? Is it not a rule you see value in?

Reply:
It make you write

```ruby
def my_method2025
```

Instead of 

```ruby
def my_method_2025
```

I don't see the value in enforcing this at all

Reply:
I've just noticed this commit 
https://github.com/DFE-Digital/publish-teacher-training/commit/d20a13274f15fa92018a5f0ef0cfae13a8ed9b0e